### DISCUSSION TITLE
Safe Exam Browser 3.1.1 on macOS Ventura (13.0.1) You are not allowed to reconfigure SEB.

### DISCUSSION THREAD

**Discussion by isabetlice** [SafeExamBrowser/seb-mac#249] [Q&A]
Although reconfiguration is permitted in the settings' exam tab, Safe Exam Browser 3.1.1 on macOS Ventura (13.0.1) continues to state that it is not permitted.

**Comment by danschlet** [comment]
Are you sure you understood the rather [complex conditions when reconfiguring is allowed](https://safeexambrowser.org/macosx/mac_usermanual_en.html#ExamPane) (depending on settings which you want to reconfigure from you may need to set the correct Reconfiguring URL)?

If you think the feature doesn't work as expected, please provide us with a testing/demo configuration so we can reproduce it.
